snuba/datasets/transactions_processor.py

```diff
@@ -137,7 +137,10 @@ def process_message(self, message, metadata) -> Optional[ProcessedMessage]:
                 ) = extract_nested(
                     measurements,
                     lambda value: float(value["value"])
-                    if isinstance(value["value"], numbers.Number)
+                    if (
+                        isinstance(value, dict)
```

Comment:
this technically only needs to check for `None` fwiw

Reply:
Please onnly check nullability as Markus said. Snuba should not do any additional sanitization of invalid values with respect to Relay.

Reply:
Done in https://github.com/getsentry/snuba/pull/1369/commits/86f506fcbbfc677711fcfca0811a196bddd6a68f